fast_neural_style/neural_style/neural_style.py

```diff
@@ -0,0 +1,231 @@
+import argparse
+import os
+import sys
+import time
+
+import numpy as np
+import torch
+from torch.autograd import Variable
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+from torchvision import datasets
+from torchvision import transforms
+
+import utils
+from transformer_net import TransformerNet
+from vgg import Vgg16
+
+
+def train(args):
+    np.random.seed(args.seed)
+    torch.manual_seed(args.seed)
+
+    if args.cuda:
+        torch.cuda.manual_seed(args.seed)
+
+    transform = transforms.Compose([
+        transforms.Scale(args.image_size),
+        transforms.CenterCrop(args.image_size),
+        transforms.ToTensor(),
+        transforms.Lambda(lambda x: x.mul(255))
```

Comment:
In my experience the current configuration works better than the normalizing at the beginning. The generator gives better results if we let it generate image with pixels in range [0, 255]. The other option is to let the generator produce image in [0, 1] range and we denormalize it before saving to file but this method gives poorer results. I tried tuning the parameter and definitely got better results with the current method.

The multiplication by 255 is needed because we need to compare the image produced by the generator with our training set images.